src/app/twiglet-graph/handleUserStateChanges.spec.ts

```diff
@@ -134,9 +135,10 @@ describe('TwigletGraphComponent:handleUserStateChanges', () => {
     });
   });
 
-  describe('textToFilterOn', () => {
-    it('greys out nodes that do not match search parameters', () => {
+  describe('textToFilterOn and filterEntities', () => {
+    it('greys out nodes that do not match search and filterEntities parameters', () => {
       response.textToFilterOn = 'first';
+      response.filterEntities = ['ent1'];
```

Comment:
This test doesn't pass without that line, which is why I added it. I thought about copy and pasting the test to check just the filterEntities, but it seemed repetitive... I still can though